### PR TITLE
feat: Change ssp master to main

### PR DESCRIPTION
**Pull request by opokornyy**
**What this PR does / why we need it**:

This PR updates SSP master to main.

**Release note**:

```
None
```

### CODE REVIEW

**Comment by lyarwood** [comment]
/hold

Thoughts on holding this until we cut the next release?

**Comment by opokornyy** [comment]
/unhold

I am unholding this since the new release is out already

File: automation/e2e-upgrade-functests/run.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ SOURCE_DIR=$(dirname "$0")
 NAMESPACE=${1:-kubevirt}
 
 RELEASE_BRANCH=${CI_BRANCH}
-if [[ -z ${RELEASE_BRANCH} ]] || [[ ${RELEASE_BRANCH} == "master" ]]
+if [[ -z ${RELEASE_BRANCH} ]] || [[ ${RELEASE_BRANCH} == "main" ]] || [[ ${RELEASE_BRANCH} == "master" ]]
```
Comment (0xFelix) [inline automation/e2e-upgrade-functests/run.sh:21]: Can this still occur?

**Review by 0xFelix — COMMENTED** [review]
/approve
/retest

File: automation/e2e-upgrade-functests/run.sh (hunk 1 above)
Comment (opokornyy) [reply automation/e2e-upgrade-functests/run.sh:21]: We need both master and main to pass tests in `openshift/release` repository, after it gets merged I will remove it in a follow up.

**Comment by lyarwood** [comment]
/hold
/retest-required

Quickly holding again as I don't want this to bork https://github.com/kubevirt/ssp-operator/pull/584

**Comment by opokornyy** [comment]
/unhold

This PR is merged - https://github.com/kubevirt/ssp-operator/pull/584

**Comment by lyarwood** [comment]
/lgtm

Let's merge this and https://github.com/openshift/release/pull/40130

**Review by 0xFelix — APPROVED** [review]
/lgtm

**Comment by akrejcir** [comment]
CI has passed.

/override tide

**Pull request merged**